README.md

```diff
@@ -1 +1,22 @@
-# setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio: setup and mobile version skeleton
+
+> This is a simple web develoment project for Microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet).
+
+## Built With
+
+- Major languages: html & css
+- Frameworks: none
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+👤 **Author1**
+
+- GitHub: [@aidan-bn](https://github.com/aidan-bn)
+- Twitter: [@aidan-bn](https://twitter.com/aidan-bn)
+- LinkedIn: [Aidan-bn](https://linkedin.com/in/aidan-bn)
```

Comment:
- [ ] Please add the remaining sections of the readme like `Contributing`, `Show your support`, `Acknowledgment`, and `License`. You can use [Microverse's readme template](https://github.com/microverseinc/readme-template) and please remember to customize its content to your project details.